src/auth/user-record.ts

```diff
@@ -180,6 +181,9 @@ export class UserRecord {
       // Ignore error.
       utils.addReadonlyGetter(this, 'customClaims', undefined);
     }
+    // Convert validSince first to UTC milliseconds and then to UTC date string.
+    utils.addReadonlyGetter(
```

Comment:
It took a while for me to realize what's going on here. Can we expand this a bit?

```
let validAfterTime: string = null;
if (typeof response.validSince !== 'undefined') {
  validAfterTime = parseDate(...);
}
utils.addReadOnlyGetter(this, 'tokensValidAfterTime', validAfterTime);
```

Reply:
Done